src/.nuget/Microsoft.NETCore.TestHost/Microsoft.NETCore.TestHost.builds

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you please diff the nuspec's for this package with the ones we currently produce to make sure the match?